packages/forma-36-react-components/src/components/Tag/Tag.css

```diff
@@ -51,3 +51,10 @@
   color: var(--color-white);
   background-color: var(--color-primary);
 }
+
+.Tag--small {
+  padding: 3px 5px;
+  font-size: 10px;
```

Comment:
Seems a bit wrong to set these values in `px` anyway.

Reply:
@domarku yeah, you are right, on my way to fix it. thx!